### PR TITLE
Update plugin.py

### PR DESCRIPTION
**Pull request by pgierz**
Allows the user to also specify kv_pairs as a list of dicts, these are transformed in the plugin to the appropriate type.

### CODE REVIEW

File: esm_pism/plugin.py [hunk 1]
```diff
@@ -6,6 +6,14 @@
 
 VALID_PISM_COUPLERS = ["ocean", "surface", "atmosphere"]
 
+def _kv_list_to_dict_of_dicts(kv_list):
+    new_dict = {}
+    for item in kv_list:
+        is not isinstance(item, dict):
```
Comment (pgierz) [inline esm_pism/plugin.py]: This needs to be if

Comment (pgierz) [inline esm_pism/plugin.py]:
```suggestion
        if not isinstance(item, dict):
```

**Comment by pgierz** [comment]
@ackerlar can you check if this bites with your changes?

**Review by ackerlar — APPROVED** [review]
Looks good to me

**Pull request merged**